avocado/utils/disk.py

```diff
@@ -123,6 +123,53 @@ def get_disks():
     return [str(disk["name"]) for disk in json_data["blockdevices"]]
 
 
+def get_all_disk_paths():
+    """
+    Returns all available disk names and alias on this  system
+
+    This will get all the sysfs disks name entries by its device
+    node name, by-uuid, by-id and by-path, irrespective of any
+    platform and device type
+
+    :returns: a list of all disk path names
+    :rtype: list of str
+    """
+    disk_list = abs_path = []
+    for path in [
+        "/dev",
+        "/dev/mapper",
+        "/dev/disk/by-id",
+        "/dev/disk/by-path",
+        "/dev/disk/by-uuid",
```

Comment:
@abdhaleegit  Just wondering if adding `by-partlabel ` `by-partuuid` will also help here?

Reply:
we can have them,  but these are available only disk/partition has valid filesystem on it.. it might change when fs changes.. there is no harm in adding.

Reply:
so can we have those added as well

Reply:
Done